port/board/afc-v3/payload.c

```diff
@@ -530,7 +530,9 @@ void clock_configuration()
     con.out14 = clock_config[14] & 0x0F;
     con.out15 = clock_config[15] & 0x0F;
 
-    adn4604_xpt_config( ADN_XPT_MAP0_CON_REG, con );
+    if (adn4604_xpt_config( ADN_XPT_MAP0_CON_REG, con )){
+        return MMC_TIMEOUT_ERR;
+    }
```

Comment:
This is not the correct way to do error propagation. You should check if `adn4604_xpt_config` returns anything different than `MMC_OK` (explicitly) and, if not, return the actual error of the function, not hardcode `MMC_TIMEOUT_ERR`.